### PR TITLE
Fix contributors format

### PR DESCRIPTION
**Pull request by MosheEichler**
## Related Issues
fixes: https://github.com/demisto/etc/issues/42299

## Description
Added untracked files to contribution-format

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/format/update_playbook.py [hunk 1]
```diff
@@ -39,14 +41,17 @@ def add_description(self):
         if 'description' not in set(self.data.keys()):
             click.secho('No description is specified for this playbook, would you like to add a description? [Y/n]',
                         fg='bright_red')
-            user_answer = ''
+            user_answer = 'y' if self.assume_yes else ''
```
Comment (reutshal) [inline demisto_sdk/commands/format/update_playbook.py:44]: was this a bug?

File: demisto_sdk/commands/init/tests/contribution_converter_test.py [hunk 2]
```diff
@@ -611,7 +611,8 @@ def test_format_converted_pack(self, contribution_converter, mocker, new_pack):
                               side_efect=self.mock_format_manager())
         contribution_converter.format_converted_pack()
 
-        assert result.call_args[1].get('prev_ver') == 'xsoar-contrib/master'
+        assert result.call_args[1].get('include_untracked')
+        assert not result.call_args[1].get('interactive')
```
Comment (reutshal) [inline demisto_sdk/commands/init/tests/contribution_converter_test.py:615]: Is this scenario still true? 
        Scenario: Formatting the added/modified files by checking against "xsoar-contrib/master" repo

Comment (reutshal) [reply demisto_sdk/commands/init/tests/contribution_converter_test.py:615]: Please update the given / when/ then of this test

**Review by reutshal — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/format/update_playbook.py (hunk 1 above)
Comment (MosheEichler) [reply demisto_sdk/commands/format/update_playbook.py:44]: Yes it was another bug https://github.com/demisto/contribution-management/runs/4724862740?check_suite_focus=true#step:12:144

**Review by reutshal — APPROVED** [review]

File: demisto_sdk/commands/init/tests/contribution_converter_test.py [hunk 3]
```diff
@@ -618,14 +618,15 @@ def test_format_converted_pack(self, contribution_converter, mocker, new_pack):
...
+        - Ensure that we format the untracked files as well and the interactive flag is set to false
         """
         contribution_converter.create_new = new_pack
         result = mocker.patch('demisto_sdk.commands.init.contribution_converter.format_manager',
                               side_efect=self.mock_format_manager())
         contribution_converter.format_converted_pack()
 
-        assert result.call_args[1].get('prev_ver') == 'xsoar-contrib/master'
+        assert result.call_args[1].get('include_untracked')
+        assert not result.call_args[1].get('interactive')
```
Comment (dorschw) [inline demisto_sdk/commands/init/tests/contribution_converter_test.py:629]:
```suggestion
        assert result.call_args[1].get('interactive') == False
```
Better be explicit, `not` might accept `None`.

**Pull request merged**